syntax = "proto3";
package greeting;

option go_package = "bikash/grpcimpl/greeting";
option java_multiple_files = true;
option java_package = " com.bikash.grpcdemo.service";
option java_outer_classname = "GreetingProto";



message HelloRequest {
  string name = 1;
  repeated string hobbies = 2;
}

message HelloResponse {
  string greeting = 1;
}

service GreetingService {
  rpc SayHello(HelloRequest) returns (HelloResponse);
}
